### ISSUE TITLE
Add alt text to card images

### ISSUE THREAD

**Issue by dtsong** [dtsong/mtg-commander-roi#25]
## Problem
Card image components may lack descriptive alt text.

## Solution
- Add descriptive alt text (card name) to all card `<Image>` elements
